### PR TITLE
[CHI-186] Unify size labels in documentation

### PR DESCRIPTION
**Pull request by davidojedacl**
@mattnickles @jllr

### CODE REVIEW

File: src/website/views/components/button.pug [hunk 1]
```diff
@@ -301,6 +301,7 @@ p.-text
 .example.-mb--3
   .-p--3
     each size in ['sm', 'md', 'lg', 'xl']
+      h6= '-'+ size
```
Comment (jllr) [inline src/website/views/components/button.pug]: @davidojedacl for being consistent we should maintain h6 or p tags for the sizes in examples and they should be inside the div (as the brand page)

File: src/website/views/components/brand.pug [hunk 2]
```diff
@@ -60,10 +60,10 @@ p.-text
 .example.-mb--3
   .-p--3(style='overflow:auto')
     each size in sizes
-      div(class=`a-col -w--12 -p--3`)
+      div(class=`a-col -w--12 -p--2`)
```
Comment (jllr) [inline src/website/views/components/brand.pug]: @davidojedacl also change -p--2 to -pb--2 in all the places where there are sizes examples

File: src/website/views/components/button.pug (hunk 1 above)
Comment (davidojedacl) [reply src/website/views/components/button.pug]: done

File: src/website/views/components/brand.pug (hunk 2 above)
Comment (davidojedacl) [reply src/website/views/components/brand.pug]: done

**Pull request merged**